nx/lib/nx.ex

```diff
@@ -12450,22 +12469,29 @@ defmodule Nx do
       [t] ->
         t
 
-      [t1 | _] = tensors ->
+      [_ | _] = tensors ->
+        [%T{vectorized_axes: vectorized_axes} | _] = tensors = broadcast_vectors(tensors)
+
+        offset = length(vectorized_axes)
+
+        [t1 | _] =
+          tensors = if vectorized_axes != [], do: Enum.map(tensors, &devectorize/1), else: tensors
```

Comment:
couldn't the first tensor not be vectorized but the other ones are?

Reply:
Not in this case, because we're checking after the broadcast_vectors

Reply:
So they all have the same vectorized axes